lib/taskana-core/src/test/java/acceptance/task/DeleteTaskCommentAccTest.java

```diff
@@ -53,18 +55,41 @@ void should_FailToDeleteTaskComment_When_UserHasNoAuthorization()
     TaskService taskService = taskanaEngine.getTaskService();
 
     List<TaskComment> taskComments =
-        taskService.getTaskComments("TKI:000000000000000000000000000000000002");
-    assertThat(taskComments).hasSize(2);
+        taskService.getTaskComments("TKI:000000000000000000000000000000000000");
+    assertThat(taskComments).hasSize(3);
 
     ThrowingCallable lambda =
-        () -> taskService.deleteTaskComment("TCI:000000000000000000000000000000000005");
+        () -> taskService.deleteTaskComment("TCI:000000000000000000000000000000000000");
 
     assertThatThrownBy(lambda).isInstanceOf(NotAuthorizedException.class);
 
     // make sure the task comment was not deleted
+    List<TaskComment> taskCommentsAfterDeletion =
+        taskService.getTaskComments("TKI:000000000000000000000000000000000000");
+    assertThat(taskCommentsAfterDeletion).hasSize(3);
+  }
+
+  @WithAccessId(user = "admin")
+  @WithAccessId(user = "taskadmin")
+  @TestTemplate
+  void should_DeleteTaskComment_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
+      throws NotAuthorizedException, TaskNotFoundException, TaskCommentNotFoundException,
+                 InvalidArgumentException, SQLException {
+
+    resetDb(false);
```

Comment:
Do we have dependencies between the tests?

Reply:
This test runs twice. It deletes a task comment which can't be deleted twice, so a reset is necessary.

Reply:
> Each invocation of a test template method behaves like the execution of a regular @Test method with full support for the same lifecycle callbacks and extensions.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-templates

Reply:
I see the issue: resetDb(..) is executed once for each TestClass.
Idea:
Depending on the current user try to delete a different ID and provide two Ids within the test-data